projects/plugins/boost/app/admin/class-config.php

```diff
@@ -53,6 +59,8 @@ public function constants() {
 			'nonces'        => Nonce::get_generated_nonces(),
 		);
 
+		$constants['autoFix'] = defined( 'AUTOFIX' ) && AUTOFIX ? true : false;
```

Comment:
Before releasing this out into the wild - let's make this more unique, so there's no collision with anything else.

Reply:
Good idea. Done!